src/main/java/com/example/draftspringboot/service/DraftService.java

```diff
@@ -26,14 +28,16 @@ public DraftResponse getById(Integer id) {
         return toModel(draftRepository.getById(id));
     }
 
-    public List<DraftResponse> getAll() {
-        return draftRepository.findAll()
-                .stream()
-                .map(this::toModel)
-                .collect(Collectors.toList());
+    public Page<DraftResponse> searchByPartName(Integer index, Integer limit, String partName) {
```

Comment:
I think that you can create `Pageable` in the controller and pass it as one parameter instead these two.